exonum-java-binding-common/src/test/java/com/exonum/binding/common/serialization/StandardSerializersRoundtripTest.java

```diff
@@ -35,7 +35,7 @@ public void roundtripStringTest() {
   }
 
   @Test
-  public void roundtripHashCodeTest() {
+  void roundtripHashCodeTest() {
```

Comment:
May use method source with Stream.of().map(HashCode::fromInt), or explicit argument converters (the latter will probably require more work), or use strings and rely on implicit conversion:

https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests-argument-conversion

Reply:
I think @ValueSource(ints = ) would be enough.